### ISSUE TITLE
Adapt Test Scenario Runner to JUnit 5

### ISSUE THREAD

**Issue by gitgabrio** [apache/incubator-kie-issues#1678]
Scenario simulation is a graphical tool used to create a
lot of "unit tests" (each of which is a "scenario") for a given DMN/DRL
model.
The tool output the test in the *.scesim format, then the backend counterpart read that file during maven test phase, and execute them.
Its current implementation relies on JUnit4 "Runner", so it has to be rewritten on top of JUnit5 version

@yesamer @pibizza

**Comment by jomarko** [comment]
Is here an impact for users that will try to open and edit old scesim files that were created before JUnit5 support ?

**Comment by gitgabrio** [comment]
Nope. The modification is only on the "maven" side: the scesim format itself (and its companion DTO) will be kept untouched
